scripts/lint/spec_lint.py

```
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)